sdk/identity/identity/src/msal/utils.ts

```diff
@@ -159,6 +160,8 @@ export class MsalBaseUtilities {
         case "endpoints_resolution_error":
           this.logger.info(formatError(scopes, error.message));
           return new CredentialUnavailableError(error.message);
+        case "device_code_polling_cancelled":
```

Comment:
How do all other credentials support cancellation? If we have to inspect error code like we are doing here for device code credential, is the plan to do the same for others eventually?

Reply:
Device code is a special case because it has to do polling. Most credentials send network requests in sequence, meaning that when there's a delay, it comes from the service, and thus cancelation through the abort-controller is feasible. We have abort-controller tests for ClientSecretCredential, ClientCertificateCredential and UsernamePasswordCredential, for example.

Reply:
Ah, sounds good. Can we add this explanation to the PR description for future reference?